client/component/people.js

```diff
@@ -54,49 +53,91 @@ function renderLayers(type, persona, profilePhotoLayers) {
     </View>
 }
 
-export function MyProfilePhoto({type='large', photo=null, faint=false, check=false, border=false}) {
+export function MyProfilePhoto({type='large', photo=null, faint=false, border=false}) {
     const personaPreview = usePersonaPreview();
     if (personaPreview?.photoUrl) {
         return <FaceImage photoUrl={photo ?? personaPreview.photoUrl} type={type} faint={faint} 
-            border={border} check={check} />
+            border={border} />
     } else if (personaPreview?.hue && personaPreview?.name) {
         return <LetterFace name={personaPreview.name} hue={personaPreview.hue} type={type} />
     } else {
         return <AnonymousFace faint={faint} type={type} border={border} />
     }
 }
 
-export function FaceImage({ face, photoUrl = null, type = 'small', faint = false, check = false, border = false}) {
-    
+export function UnverifiedBadge({size = 14}) {
+    const s = UnverifiedBadgeStyles;
+    return (
+        <View style={[s.container, { width: size, height: size, borderRadius: size / 2 }]}>
+            <Text style={[s.text, { fontSize: size === 24 ? 18 : 12 }]}>?</Text>
+        </View>
+    );
+}
+
+const UnverifiedBadgeStyles = StyleSheet.create({
+    container: {
+        backgroundColor: colorGreyHover,
+        justifyContent: 'center',
+        alignItems: 'center',
+    },
+    text: {
+        color: 'black',
+    },
+});
+
+function Badge({type, name}) {
+    const datastore = useDatastore();
+    const fbUser = datastore.getFirebaseUser();
+    const personaPreview = usePersonaPreview();
+    const verificationStatus = personaPreview?.verificationStatus ?? null;
```

Comment:
As written, it seems the badge will always be showing the verification status of the user who is logged in. Don't we actually want it to be showing the verification status of the user who we are showing a profile photo for?

Reply:
thanks for catching this. fixed.